src/main/proto/netflix/titus/validator/validator.proto

```diff
@@ -95,10 +95,18 @@ message SecurityGroupValidationRequest {
     // * a security group id (for example sg-12345678)
     // * a security group ARN
     // * a security group name/alias (for example 'titus-sg')
-    string securityGroup = 1;
+    // Use securityGroups instead
+    string securityGroup = 1 [deprecated = true];
```

Comment:
Just remove these fields if you're not using them anymore. No need to go through formal deprecation